src/test/java/org/junitpioneer/jupiter/RetryingTestExtensionTests.java

```diff
@@ -126,6 +203,51 @@ void skipByAssumption() {
 			Assumptions.assumeFalse(true);
 		}
 
+		@RetryingTest(maxAttempts = 4, minSuccess = 2)
+		void executesTwice() {
+		}
+
+		@RetryingTest(maxAttempts = 4, minSuccess = 2)
+		void executesTwiceWithTwoFails() {
+			EXECUTION_COUNT++;
+			if (EXECUTION_COUNT == 2 || EXECUTION_COUNT == 3) {
+				throw new IllegalArgumentException();
+			}
+		}
+
+		@RetryingTest(maxAttempts = 4, minSuccess = 2)
+		void executesOnceWithThreeFails() {
+			EXECUTION_COUNT++;
+			if (EXECUTION_COUNT != 2) {
+				throw new IllegalArgumentException();
+			}
+		}
+
+		@RetryingTest(maxAttempts = 4, minSuccess = 2)
+		void failsThreeTimes() {
+			throw new IllegalArgumentException();
+		}
+
+		@RetryingTest
+		void missingMaxAttempts() {
+			// Do nothing
+		}
+
+		@RetryingTest(value = 1, maxAttempts = 1)
+		void maxAttemptsAndValueSet() {
+			// Do nothing
+		}
+
+		@RetryingTest(maxAttempts = 1, minSuccess = 1)
+		void maxAttemptsEqualsMinSuccess() {
+			// Do nothing
+		}
+
+		@RetryingTest(maxAttempts = 2, minSuccess = 0)
+		void minSuccessLessThanOne() {
+			// Do nothing
+		}
+
```

Comment:
More tests:
* `maxAttempts` <= 0
* `maxAttempts` < `minSuccess`

Reply:
The second suggested test should be covered by `maxAttemptsEqualsMinSuccess`. The condition is that `maxAttempts` <= `minSuccess`. I added the requested test though.